[I 2021-04-05 02:33:37.145 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-05 02:33:37.538 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-05 02:33:37.538 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-05 02:33:37.539 ServerApp] nbclassic | extension was successfully linked.
[W 2021-04-05 02:33:37.603 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-05 02:33:37.605 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-05 02:33:37.605 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-05 02:33:37.609 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-05 02:33:37.616 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-05 02:33:37.617 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-05 02:33:37.617 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-05 02:33:37.617 ServerApp] http://localhost:8888/lab?token=788873007795dbaebe6b90f49f226478caa46eba12060e99
[I 2021-04-05 02:33:37.617 ServerApp]  or http://127.0.0.1:8888/lab?token=788873007795dbaebe6b90f49f226478caa46eba12060e99
[I 2021-04-05 02:33:37.617 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-05 02:33:37.635 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-40449-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/lab?token=788873007795dbaebe6b90f49f226478caa46eba12060e99
     or http://127.0.0.1:8888/lab?token=788873007795dbaebe6b90f49f226478caa46eba12060e99
[I 2021-04-05 02:33:42.197 LabApp] Build is up to date
[W 2021-04-05 02:33:43.100 ServerApp] 404 GET /api/contents/demo.ipynb?content=0&1617604423092 (::1): No such file or directory: demo.ipynb
[W 2021-04-05 02:33:43.100 ServerApp] No such file or directory: demo.ipynb
[W 2021-04-05 02:33:43.100 ServerApp] 404 GET /api/contents/demo.ipynb?content=0&1617604423092 (::1) 2.83ms referer=http://localhost:8888/lab
[W 2021-04-05 02:33:43.101 ServerApp] 404 GET /api/contents/inference.ipynb?content=0&1617604423093 (::1): No such file or directory: inference.ipynb
[W 2021-04-05 02:33:43.101 ServerApp] No such file or directory: inference.ipynb
[W 2021-04-05 02:33:43.102 ServerApp] 404 GET /api/contents/inference.ipynb?content=0&1617604423093 (::1) 3.58ms referer=http://localhost:8888/lab
-\|/-\|/-\|/-[I 2021-04-05 02:33:53.202 ServerApp] Kernel started: 6d81f1a6-58f9-4eb8-805a-40bdb312446f
[W 2021-04-05 02:33:56.786 ServerApp] Got events for closed stream None
[I 2021-04-05 02:34:32.263 ServerApp] Saving file at /machine_learning/preprocessing_voice/play_sound.ipynb
[I 2021-04-05 02:35:24.498 ServerApp] Saving file at /machine_learning/preprocessing_voice/play_sound.ipynb
[I 2021-04-05 02:35:25.750 ServerApp] Starting buffering for 6d81f1a6-58f9-4eb8-805a-40bdb312446f:4a0e78e1-625a-41d2-8e51-1a6e6c45f8d7
[I 2021-04-05 02:42:40.890 ServerApp] Shutting down on /api/shutdown request.
[I 2021-04-05 02:42:40.890 ServerApp] Shutting down 1 kernel
[I 2021-04-05 02:42:41.097 ServerApp] Kernel shutdown: 6d81f1a6-58f9-4eb8-805a-40bdb312446f
[I 2021-04-07 15:59:39.101 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-07 15:59:39.533 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-07 15:59:39.533 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-07 15:59:39.533 ServerApp] nbclassic | extension was successfully linked.
[W 2021-04-07 15:59:39.590 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-07 15:59:39.591 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-07 15:59:39.591 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-07 15:59:39.595 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-07 15:59:39.601 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-07 15:59:39.601 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-07 15:59:39.601 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-07 15:59:39.601 ServerApp] http://localhost:8888/lab?token=d766295e08c097fda522e0909ddc7e5685106b4e1235889e
[I 2021-04-07 15:59:39.601 ServerApp]  or http://127.0.0.1:8888/lab?token=d766295e08c097fda522e0909ddc7e5685106b4e1235889e
[I 2021-04-07 15:59:39.601 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-07 15:59:39.615 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-8130-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/lab?token=d766295e08c097fda522e0909ddc7e5685106b4e1235889e
     or http://127.0.0.1:8888/lab?token=d766295e08c097fda522e0909ddc7e5685106b4e1235889e
[I 2021-04-07 15:59:43.859 LabApp] Build is up to date
[W 2021-04-07 15:59:44.739 ServerApp] 404 GET /api/contents/ch03.ipynb?content=0&1617825584733 (::1): No such file or directory: ch03.ipynb
[W 2021-04-07 15:59:44.739 ServerApp] No such file or directory: ch03.ipynb
[W 2021-04-07 15:59:44.740 ServerApp] 404 GET /api/contents/ch03.ipynb?content=0&1617825584733 (::1) 1.79ms referer=http://localhost:8888/lab
-\|/-\|/-\|/-\[I 2021-04-07 15:59:55.252 ServerApp] Kernel started: 858d135a-ef10-4283-a2ac-cf12a8d03c90
[W 2021-04-07 15:59:58.819 ServerApp] Got events for closed stream None
[I 2021-04-07 16:00:17.608 ServerApp] Kernel started: 6cdff588-fbf3-407d-9505-4ec170c903a2
[I 2021-04-07 16:00:19.541 ServerApp] Copying scripting/bash.ipynb to /scripting
[I 2021-04-07 16:00:37.647 ServerApp] Kernel started: 3b229ab0-ed1b-40ba-b291-a9c27244ec05
[I 2021-04-07 16:01:33.666 ServerApp] Saving file at /scripting/movgif.ipynb
[I 2021-04-07 16:01:35.213 ServerApp] Starting buffering for 3b229ab0-ed1b-40ba-b291-a9c27244ec05:8917d576-9644-4d52-8b91-38b0fb2ba17c
[I 2021-04-07 16:01:36.619 ServerApp] Saving file at /scripting/bash.ipynb
[I 2021-04-07 16:01:37.140 ServerApp] Starting buffering for 6cdff588-fbf3-407d-9505-4ec170c903a2:81117ecd-3562-4e81-90da-eab2ffea4e36
[I 2021-04-07 16:01:42.392 ServerApp] Starting buffering for 858d135a-ef10-4283-a2ac-cf12a8d03c90:8543abd8-19bb-44e1-83d8-4b9fbc2ab726
[I 2021-04-08 18:50:39.228 ServerApp] Shutting down on /api/shutdown request.
[I 2021-04-08 18:50:39.233 ServerApp] Shutting down 3 kernels
[I 2021-04-08 18:50:39.545 ServerApp] Kernel shutdown: 858d135a-ef10-4283-a2ac-cf12a8d03c90
[I 2021-04-08 18:50:39.547 ServerApp] Kernel shutdown: 6cdff588-fbf3-407d-9505-4ec170c903a2
[I 2021-04-08 18:50:39.548 ServerApp] Kernel shutdown: 3b229ab0-ed1b-40ba-b291-a9c27244ec05
[I 2021-04-13 15:57:59.644 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-13 15:58:00.000 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-13 15:58:00.000 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-13 15:58:00.001 ServerApp] nbclassic | extension was successfully linked.
[I 2021-04-13 15:58:00.050 ServerApp] The port 8888 is already in use, trying another port.
[I 2021-04-13 15:58:00.051 ServerApp] The port 8889 is already in use, trying another port.
[I 2021-04-13 15:58:00.051 ServerApp] The port 8890 is already in use, trying another port.
[W 2021-04-13 15:58:00.058 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-13 15:58:00.060 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-13 15:58:00.060 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-13 15:58:00.064 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-13 15:58:00.070 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-13 15:58:00.070 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-13 15:58:00.070 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-13 15:58:00.070 ServerApp] http://localhost:8891/lab?token=1f2e06b1fb614e2d39e395f51940e118d565ec7ddf7d6a86
[I 2021-04-13 15:58:00.070 ServerApp]  or http://127.0.0.1:8891/lab?token=1f2e06b1fb614e2d39e395f51940e118d565ec7ddf7d6a86
[I 2021-04-13 15:58:00.070 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-13 15:58:00.087 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-71430-open.html
    Or copy and paste one of these URLs:
        http://localhost:8891/lab?token=1f2e06b1fb614e2d39e395f51940e118d565ec7ddf7d6a86
     or http://127.0.0.1:8891/lab?token=1f2e06b1fb614e2d39e395f51940e118d565ec7ddf7d6a86
[I 2021-04-13 15:58:04.590 LabApp] Build is up to date
[W 2021-04-13 15:58:05.907 ServerApp] 404 GET /api/contents/test_toxic_models.ipynb?content=0&1618343885901 (::1): No such file or directory: test_toxic_models.ipynb
[W 2021-04-13 15:58:05.907 ServerApp] No such file or directory: test_toxic_models.ipynb
[W 2021-04-13 15:58:05.907 ServerApp] 404 GET /api/contents/test_toxic_models.ipynb?content=0&1618343885901 (::1) 2.11ms referer=http://localhost:8891/lab
[W 2021-04-13 15:58:05.909 ServerApp] 404 GET /api/contents/helpers/utils.py?content=0&1618343885903 (::1): No such file or directory: helpers/utils.py
[W 2021-04-13 15:58:05.910 ServerApp] No such file or directory: helpers/utils.py
[W 2021-04-13 15:58:05.910 ServerApp] 404 GET /api/contents/helpers/utils.py?content=0&1618343885903 (::1) 1.65ms referer=http://localhost:8891/lab
-\|/-[W 2021-04-13 15:58:06.444 ServerApp] 404 GET /api/contents/helpers?1618343886440 (::1): No such file or directory: helpers
[W 2021-04-13 15:58:06.444 ServerApp] No such file or directory: helpers
[W 2021-04-13 15:58:06.445 ServerApp] 404 GET /api/contents/helpers?1618343886440 (::1) 2.10ms referer=http://localhost:8891/lab
\|/-[I 2021-04-13 15:59:15.726 ServerApp] Kernel started: 43037573-3ab4-41da-83eb-577e66a3a31a
[W 2021-04-13 15:59:19.707 ServerApp] Got events for closed stream None
[I 2021-04-13 15:59:25.016 ServerApp] Saving file at /machine_learning/preprocessing_voice/ffmpeg.ipynb
[I 2021-04-13 15:59:26.031 ServerApp] Starting buffering for 43037573-3ab4-41da-83eb-577e66a3a31a:0f582db6-2f66-452b-9d00-36aeaefc6ea0
[I 2021-04-16 15:54:42.402 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-16 15:54:42.833 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-16 15:54:42.833 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-16 15:54:42.833 ServerApp] nbclassic | extension was successfully linked.
[W 2021-04-16 15:54:42.901 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-16 15:54:42.903 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-16 15:54:42.903 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-16 15:54:42.906 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-16 15:54:42.913 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-16 15:54:42.913 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-16 15:54:42.913 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-16 15:54:42.913 ServerApp] http://localhost:8888/lab?token=34bde45ede6a5ce3cb2e15902d8424eb64649f917c0817b4
[I 2021-04-16 15:54:42.913 ServerApp]  or http://127.0.0.1:8888/lab?token=34bde45ede6a5ce3cb2e15902d8424eb64649f917c0817b4
[I 2021-04-16 15:54:42.913 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-16 15:54:42.930 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-43358-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/lab?token=34bde45ede6a5ce3cb2e15902d8424eb64649f917c0817b4
     or http://127.0.0.1:8888/lab?token=34bde45ede6a5ce3cb2e15902d8424eb64649f917c0817b4
[I 2021-04-16 15:54:47.597 LabApp] Build is up to date
[W 2021-04-16 15:54:48.991 ServerApp] 404 GET /api/contents/lab_Nsight.ipynb?content=0&1618602888985 (::1): No such file or directory: lab_Nsight.ipynb
[W 2021-04-16 15:54:48.991 ServerApp] No such file or directory: lab_Nsight.ipynb
[W 2021-04-16 15:54:48.991 ServerApp] 404 GET /api/contents/lab_Nsight.ipynb?content=0&1618602888985 (::1) 1.84ms referer=http://localhost:8888/lab
-\|/-\|/-[I 2021-04-16 15:54:58.369 ServerApp] Kernel started: 37d372a2-58c7-47ba-9458-56b7eb8eff9a
[W 2021-04-16 15:55:02.026 ServerApp] Got events for closed stream None
[I 2021-04-16 15:55:20.846 ServerApp] Saving file at /machine_learning/preprocessing_voice/total_audio.ipynb
[I 2021-04-16 16:11:33.340 ServerApp] Saving file at /machine_learning/preprocessing_voice/total_audio.ipynb
[I 2021-04-16 16:11:36.998 ServerApp] Saving file at /machine_learning/preprocessing_voice/total_audio.ipynb
[I 2021-04-16 16:11:38.080 ServerApp] Starting buffering for 37d372a2-58c7-47ba-9458-56b7eb8eff9a:61d10f87-1938-4d8f-b33b-5f231187425e
[I 2021-04-16 17:44:27.297 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-16 17:44:27.749 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-16 17:44:27.749 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-16 17:44:27.749 ServerApp] nbclassic | extension was successfully linked.
[I 2021-04-16 17:44:27.811 ServerApp] The port 8888 is already in use, trying another port.
[W 2021-04-16 17:44:27.820 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-16 17:44:27.822 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-16 17:44:27.822 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-16 17:44:27.826 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-16 17:44:27.833 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-16 17:44:27.834 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-16 17:44:27.834 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-16 17:44:27.834 ServerApp] http://localhost:8889/lab?token=6306616cb410a31b9dec17f7c248f0d4e6296b955f314b4d
[I 2021-04-16 17:44:27.834 ServerApp]  or http://127.0.0.1:8889/lab?token=6306616cb410a31b9dec17f7c248f0d4e6296b955f314b4d
[I 2021-04-16 17:44:27.834 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-16 17:44:27.854 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-44348-open.html
    Or copy and paste one of these URLs:
        http://localhost:8889/lab?token=6306616cb410a31b9dec17f7c248f0d4e6296b955f314b4d
     or http://127.0.0.1:8889/lab?token=6306616cb410a31b9dec17f7c248f0d4e6296b955f314b4d
[I 2021-04-16 17:44:34.814 LabApp] Build is up to date
-\|/-\|[I 2021-04-16 17:44:36.921 ServerApp] Kernel started: 49b9afb5-5ae0-405a-81e7-af50ed0ddd81
/-\|/-[W 2021-04-16 17:44:40.910 ServerApp] Got events for closed stream None
[I 2021-04-16 17:44:48.116 ServerApp] Kernel started: 0a933808-30c0-42a1-928e-da27d34dea90
[I 2021-04-16 17:45:04.039 ServerApp] Saving file at /scripting/bash.ipynb
[I 2021-04-16 18:17:07.968 ServerApp] Starting buffering for 0a933808-30c0-42a1-928e-da27d34dea90:c866cae3-5c46-4511-b392-5d9acb657eae
[I 2021-04-16 18:17:07.970 ServerApp] Starting buffering for 49b9afb5-5ae0-405a-81e7-af50ed0ddd81:4de3a318-5157-4905-9b9f-55ccd289d33f
[I 2021-04-17 08:39:31.918 ServerApp] jupyterlab | extension was successfully linked.
[I 2021-04-17 08:39:32.444 ServerApp] jupyter_nbextensions_configurator | extension was found and enabled by nbclassic. Consider moving the extension to Jupyter Server's extension paths.
[I 2021-04-17 08:39:32.444 ServerApp] jupyter_nbextensions_configurator | extension was successfully linked.
[I 2021-04-17 08:39:32.444 ServerApp] nbclassic | extension was successfully linked.
[I 2021-04-17 08:39:32.499 ServerApp] The port 8888 is already in use, trying another port.
[I 2021-04-17 08:39:32.500 ServerApp] The port 8889 is already in use, trying another port.
[W 2021-04-17 08:39:32.507 ServerApp] jupyter_nbextensions_configurator | extension failed loading with message: 'nbextensions_path'
[I 2021-04-17 08:39:32.508 LabApp] JupyterLab extension loaded from /Users/othrif/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyterlab
[I 2021-04-17 08:39:32.508 LabApp] JupyterLab application directory is /Users/othrif/.pyenv/versions/3.8.5/share/jupyter/lab
[I 2021-04-17 08:39:32.511 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-04-17 08:39:32.518 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-04-17 08:39:32.518 ServerApp] Serving notebooks from local directory: /Users/othrif/github/notes/content
[I 2021-04-17 08:39:32.518 ServerApp] Jupyter Server 1.4.1 is running at:
[I 2021-04-17 08:39:32.519 ServerApp] http://localhost:8890/lab?token=116839f50e3ec54eb36d42780836a00b0f6b1f2a7e0322c8
[I 2021-04-17 08:39:32.519 ServerApp]  or http://127.0.0.1:8890/lab?token=116839f50e3ec54eb36d42780836a00b0f6b1f2a7e0322c8
[I 2021-04-17 08:39:32.519 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-04-17 08:39:32.537 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/othrif/Library/Jupyter/runtime/jpserver-45733-open.html
    Or copy and paste one of these URLs:
        http://localhost:8890/lab?token=116839f50e3ec54eb36d42780836a00b0f6b1f2a7e0322c8
     or http://127.0.0.1:8890/lab?token=116839f50e3ec54eb36d42780836a00b0f6b1f2a7e0322c8
[I 2021-04-17 08:39:37.728 LabApp] Build is up to date
-\|/[I 2021-04-17 08:39:39.327 ServerApp] Kernel started: c3c7858d-814e-4690-a80e-a0eb4af506b1
[I 2021-04-17 08:39:39.349 ServerApp] Kernel started: 37f617a9-3ede-4945-b31e-4ff55a9913bf
-\|/-\[I 2021-04-17 08:39:48.349 ServerApp] Kernel started: 1b360f92-0fc1-4b4c-8f7a-f9e96f9eda50
[I 2021-04-17 08:39:51.649 ServerApp] Copying python/basics/hashing.ipynb to /python/basics
[I 2021-04-17 08:39:53.710 ServerApp] Kernel started: cfdcc9ad-424d-49ce-9806-2a2575c5025d
[I 2021-04-17 08:39:55.332 ServerApp] Starting buffering for c3c7858d-814e-4690-a80e-a0eb4af506b1:d7405b65-e4a2-43a9-a2ac-774b7e882e78
[I 2021-04-17 08:39:55.878 ServerApp] Starting buffering for 1b360f92-0fc1-4b4c-8f7a-f9e96f9eda50:4e671aa2-5402-4f69-9fa9-5095d692b2e6
[I 2021-04-17 08:41:53.602 ServerApp] Saving file at /python/basics/map.ipynb
[I 2021-04-17 08:43:11.511 ServerApp] Saving file at /python/basics/map.ipynb
[I 2021-04-17 08:43:26.110 ServerApp] Copying python/basics/map.ipynb to /python/basics
[W 2021-04-17 08:43:31.545 ServerApp] delete /python/basics/map-Copy1.ipynb
[I 2021-04-17 08:45:11.545 ServerApp] Saving file at /python/basics/map.ipynb
[I 2021-04-17 08:47:11.583 ServerApp] Saving file at /python/basics/map.ipynb
[I 2021-04-17 08:49:11.620 ServerApp] Saving file at /python/basics/map.ipynb
[I 2021-04-17 08:58:05.828 ServerApp] Starting buffering for cfdcc9ad-424d-49ce-9806-2a2575c5025d:bc5faa0c-55f1-4af7-91fd-d3577deeaf4e
[I 2021-04-17 08:58:05.829 ServerApp] Starting buffering for 37f617a9-3ede-4945-b31e-4ff55a9913bf:f47211a2-1764-4fbc-afea-f47dcba273ff
